pom.xml

```diff
@@ -168,7 +168,7 @@
           </plugins>
         </build>
         <modules>
-           <module>kernel</module>
+           <!--<module>kernel</module>-->
```

Comment:
Why are we commenting the kernel module ?

Reply:
we dont have any change in this patch(1.1.5.3, we have changes only in packetmanager